### ISSUE TITLE
First step. React from tutorial [Main task]

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/react-only-13#1]
https://medium.com/quick-code/lets-learn-react-chapter-1-setting-up-environment-e9505b6644

after reading each chapter you need to push your code updates here and also give me a short resume about what exactly did you. it'll needed for better understanding your progress

**Comment by khyati0212** [comment]
@atherdon so this is for me?

**Comment by atherdon** [comment]
@khyati0212  yep

**Comment by khyati0212** [comment]
So react is related to ES6?

**Comment by atherdon** [comment]
yep. create react app also have included Jest package

**Comment by khyati0212** [comment]
yeah I saw that in many tutorial. I will learn React too now. Interesting. :) I will keep you updated. I really liked this way of learning.

**Comment by atherdon** [comment]
good. react have a big promise at js field. we also will use it soon on my projects, so it's a good thing in all ways
